ci-operator/config/openshift-knative/eventing-kafka/openshift-knative-eventing-kafka-release-next__48.yaml

```diff
@@ -250,8 +242,6 @@ tests:
         name: knative-eventing-kafka-mtcontroller
       - env: KNATIVE_EVENTING_KAFKA_MTRECEIVE_ADAPTER
         name: knative-eventing-kafka-mtreceive-adapter
-      - env: KNATIVE_EVENTING_KAFKA_POST_INSTALL
```

Comment:
I think I'd name that than `KNATIVE_EVENTING_KAFKA_CHANNEL_POST_INSTALL` ?

Reply:
that was removed anyway.